.vim/ftplugin/python.vim

```diff
@@ -1,5 +1,23 @@
 "Highlight real tabs"
-highlight Tab guibg=3 ctermbg=3
+"Colors -> guibg={color-name} ctermgb={color-number}
+"NR-16   NR-8    COLOR NAME ~
+"0	     0	     Black
+"1	     4	     DarkBlue
+"2	     2	     DarkGreen
+"3	     6	     DarkCyan
+"4	     1	     DarkRed
+"5	     5	     DarkMagenta
+"6	     3	     Brown, DarkYellow
+"7	     7	     LightGray, LightGrey, Gray, Grey
+"8	     0*	     DarkGray, DarkGrey
+"9	     4*	     Blue, LightBlue "10	     2*	     Green, LightGreen
+"11	     6*	     Cyan, LightCyan
+"12	     1*	     Red, LightRed
+"13	     5*	     Magenta, LightMagenta
+"14	     3*	     Yellow, LightYellow
+"15	     7*	     White
+
```

Comment:
I didn't understand why you added all of this comments. Is it just for user documentation?